listenbrainz/webserver/static/js/src/follow/UserSocialNetwork.tsx

```diff
@@ -0,0 +1,154 @@
+import * as React from "react";
+import { includes as _includes } from "lodash";
+
+import APIService from "../APIService";
+import UserListModalEntry from "./UserListModalEntry";
+import FollowerFollowingModal from "./FollowerFollowingModal";
+import SimilarUsersModal from "./SimilarUsersModal";
+
+export type UserSocialNetworkProps = {
+  user: ListenBrainzUser;
+  loggedInUser: ListenBrainzUser | null;
+};
+
+type UserSocialNetworkState = {
+  followerList: Array<ListenBrainzUser>;
+  followingList: Array<ListenBrainzUser>;
+  similarUsersList: Array<SimilarUser>;
+};
+
+export default class UserSocialNetwork extends React.Component<
+  UserSocialNetworkProps,
+  UserSocialNetworkState
+> {
+  APIService: APIService;
+
+  constructor(props: UserSocialNetworkProps) {
+    super(props);
+    this.APIService = new APIService(`${window.location.origin}/1`);
+    this.state = {
+      followerList: [],
+      followingList: [],
+      similarUsersList: [],
+    };
+
+    // DO NOT CHANGE THIS ORDER
+    // Otherwise react messes up the follow button props;
+    // The way to fix this is to not pass the loggedInUserFollowsUser prop
+    // into the FollowButton compoent.
```

Comment:
Could the issue you mention be fixed in #1328 ?

Reply:
It seems to be fixed now